### PR TITLE
Add member to sig-docs-ko-approver

### PR DESCRIPTION
**Pull request by ysyukr**
Add @ysyukr to sig-docs-ko-approver.

related with kubernetes/website#20094

### CODE REVIEW

**Comment by sftim** [comment]
Based on https://github.com/kubernetes/website/pull/20094
/lgtm

File: config/kubernetes/sig-docs/teams.yaml [hunk 1]
```diff
@@ -339,6 +341,7 @@ teams:
     - gochist
     - hanjiayao
     - huynguyennovem
+    - ianychoi
```
Comment (sftim) [inline config/kubernetes/sig-docs/teams.yaml:344]: This is based on https://github.com/kubernetes/website/blob/8f7f707d6b9ab27b393638134729f5ced94f2818/OWNERS_ALIASES#L149 (AIUI)

Comment (zacharysarah) [reply config/kubernetes/sig-docs/teams.yaml:344]: @ysyukr Please limit this PR only to yourself. Other members can add themselves to various groups as need be.

Comment (ysyukr) [reply config/kubernetes/sig-docs/teams.yaml]: @zacharysarah I agree with you. I have updated this PR to be registered only with me.

**Comment by ysyukr** [comment]
@zacharysarah It has been about 2 days since I updated this PR and modified it to register only for me, but the review has not been made.

File: config/kubernetes/sig-docs/teams.yaml [hunk 2]
```diff
@@ -317,6 +318,7 @@ teams:
     - truongnh1992 # L10n: Vietnamese
     - vineethreddy02 # Docs 1.18 lead
     - xichengliudui # L10n: Chinese
+    - ysyukr # L10n: Korean
```
Comment (zacharysarah) [inline config/kubernetes/sig-docs/teams.yaml:321]: @gochist 👋 Can you please verify whether you need additional members with write access?

Comment (ysyukr) [reply config/kubernetes/sig-docs/teams.yaml:321]: @zacharysarah Thanks for the comments.
I'll discuss this part within the localization team, including myself. and i will modify the PR by excluding this part. after discussion, i will open PR as needed.

Comment (ysyukr) [reply config/kubernetes/sig-docs/teams.yaml]: > @gochist 👋 Can you please verify whether you need additional members with write access?

@zacharysarah Thanks for the comments.
I'll discuss this part within the localization team, including myself. and i will modify the PR by excluding this part. after discussion, i will open PR as needed.

**Comment by ysyukr** [comment]
@zacharysarah Thanks for the comments.
I'll discuss this part within the localization team, including myself. and i will modify the PR by excluding this part. after discussion, i will open PR as needed.

**Comment by ysyukr** [comment]
@zacharysarah Request a review for modified pr.

**Pull request merged**